[2023-09-04T06:02:22.822+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [queued]>
[2023-09-04T06:02:22.837+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [queued]>
[2023-09-04T06:02:22.838+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:22.860+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-20 00:00:00+00:00
[2023-09-04T06:02:22.869+0000] {standard_task_runner.py:57} INFO - Started process 5307 to run task
[2023-09-04T06:02:22.872+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-03-20T00:00:00+00:00', '--job-id', '420', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp9aagfols']
[2023-09-04T06:02:22.874+0000] {standard_task_runner.py:85} INFO - Job 420: Subtask insert_into_table
[2023-09-04T06:02:22.947+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:23.082+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-20T00:00:00+00:00'
[2023-09-04T06:02:23.084+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-20', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:23.096+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:23.108+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:23.128+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-20', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:23.132+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:23.147+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220320T000000, start_date=20230904T060222, end_date=20230904T060223
[2023-09-04T06:02:23.206+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:23.224+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:03.390+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [queued]>
[2023-09-04T06:05:03.407+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [queued]>
[2023-09-04T06:05:03.407+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:03.434+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-20 00:00:00+00:00
[2023-09-04T06:05:03.444+0000] {standard_task_runner.py:57} INFO - Started process 6563 to run task
[2023-09-04T06:05:03.449+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-03-20T00:00:00+00:00', '--job-id', '958', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpkc0ch7mf']
[2023-09-04T06:05:03.454+0000] {standard_task_runner.py:85} INFO - Job 958: Subtask insert_into_table
[2023-09-04T06:05:03.533+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-03-20T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:03.660+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-20T00:00:00+00:00'
[2023-09-04T06:05:03.661+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-20', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:03.675+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:03.687+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:03.704+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-20', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:03.708+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:03.723+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220320T000000, start_date=20230904T060503, end_date=20230904T060503
[2023-09-04T06:05:03.781+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:03.804+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
